dataobj/schedule.h

```diff
@@ -169,12 +171,12 @@ class schedule_t
 	/**
 	 * Inserts a coordinate at current_stop into the schedule.
 	 */
-	bool insert(const grund_t* gr, uint8 minimum_loading = 0, uint16 waiting_time_shift = 0, uint8 coupling_point = 0);
+	bool insert(const grund_t* gr, uint8 minimum_loading = 0, uint16 waiting_time_shift = 0, uint8 coupling_point = 0, uint8 reverse_flag = 0);
```

Comment:
[Q] For what is `reverse_flag` used? It seems that nobody passes this value explicitly.